src/ansys/motorcad/core/methods/adaptive_geometry.py

```diff
@@ -192,3 +192,36 @@ def delete_region(self, region, remove_children=False):
         method = "DeleteRegion"
         params = [raw_region, remove_children]
         self.connection.send_and_receive(method, params)
+
+    def subtract_region(self, region, region_subtract):
+        """Subtract a single region from another region.
+
+        Parameters
+        ----------
+        region : ansys.motorcad.core.geometry.Region
+            Motor-CAD region object
+
+        region_subtract : ansys.motorcad.core.geometry.Region
```

Comment:
Can you improve this docstring e.g. Motor-CAD region that is subtracted from region